content/docs/command-reference/remote/add.md

```diff
@@ -140,7 +140,8 @@ $ dvc remote modify myremote endpointurl \
                     https://object-storage.example.com
 ```
 
-> See `dvc remote modify` for a full list of S3 API parameters.
+> See `dvc remote modify`(the "Amazon S3" section) for a full list of S3 API
+> parameters.
```

Comment:
We can leave this as it was. https://dvc.org/doc/command-reference/remote/modify#available-parameters-per-storage-type has a section called "Click for S3 API compatible storage", matching the one here. The user can decide which one to check (or both).